src/pages/UserProfile/SelfProfile/SelfProfilePage.tsx

```diff
@@ -26,7 +26,7 @@ export default function SelfProfilePage(): JSX.Element {
   }
 
   return (
-    <div className="max-w-5xl mx-auto">
+    <div className="page-frame pb-5 max-w-5xl">
```

Comment:
Instead of having to include additional bottom padding here and in profile-settings page, can you modify page-frame to include a standard bottom padding for all pages.